### DISCUSSION TITLE
Vue des infrastructures logiques.

### DISCUSSION THREAD

**Discussion by SteevensICam** [dbarzin/mercator#477] [Q&A]
Bonjour,
Nous venons de déployer la solution, nous sommes nouveaux sur l'utilisation de votre outil.
Nous rencontrons un problème au niveau de la Vue des infrastructures logiques au moment de l'affichage d'un réseau logique.
La création et la modification de la fiche réseau se passe bien, mais pour l'affichage, nous avons un message d'erreur : 
![Capture d’écran 2023-08-22 091739](https://github.com/dbarzin/mercator/assets/142876160/ada96680-5215-4214-8e26-6c0bb341b0fc)
Sommes-nous les seuls à disposer de ce message d'erreur ?
Je vous en remercie.

**Comment by dbarzin** [comment]
Pouvez-vous me confirmer que vous utilisez la version Docker avec sqlite ?
Sur quelle page avez-vous cette erreur (URL) ?

**Reply by SteevensICam** [reply]
Nous avons installé la version docker avec la version sqlite backend en version donnée persistante.
Pour accéder à la page : dans le menu, cliquer sur "vue des infrastructures" logiques, réseaux, et cliquer sur le nom du réseau ou à partir du bouton afficher --> on arrive sur la page de visualisation du réseau. https://domaine/admin/networks/1
![Capture d’écran 2023-08-22](https://github.com/dbarzin/mercator/assets/142876160/ec126e31-8a2d-4edd-a2af-050e0dc57233)

**Reply by SteevensICam** [reply]
Bonjour, avez-vous réussi à reproduire le problème sur votre environnement ? Je vous en remercie.

**Comment by dbarzin** [comment]
C'est corrigé !

**Reply by SteevensICam** [reply]
Bonjour, 
je vous en remercie, c'est corrigé.
Par contre, quand je clique dans le menu de gauche, sur "vue de l'administration" et ensuite sur "services d'annuaire d'administration" j'ai un message d'erreur, que nous n'avions pas avant l'upgrade de version : 

![Capture d’écran 2023-08-28 083856](https://github.com/dbarzin/mercator/assets/142876160/169c6770-5d5f-4093-ba72-f4af9ef6fc29)

**Comment by dbarzin** [comment]
C'est corrigé !
Merci,
Didier

**Reply by SteevensICam** [reply]
Bonjour, 
Je vous en remercie, mais j'ai une autre erreur maintenant, toujours sur le même lien. 
La correction a dû générer un autre problème : 
![Capture d’écran 2023-08-28 171726](https://github.com/dbarzin/mercator/assets/142876160/247fa656-30f2-44a0-a829-b4306c9d1b6e)

**Comment by dbarzin** [comment]
OK, désole. C'est corrigé.
Merci,
Didier

**Reply by SteevensICam** [reply]
Bonjour, 
Je vous confirme que tt est ok, je vous en remercie.
Bonne journée.
